### PR TITLE
Define whether a test is required by code

### PR DESCRIPTION
**Pull request by yejianquan**
#### Why I did it
Define whether a test job is required before merging by code.
Let the failure of multi-asic and t0-sonic don't block pr merge.
The 'required' configuration can be modified by the owner in the future.
#### How I did it
Required:
t1-lag, t0
Not required:
multi-asic, t0-sonic
#### How to verify it
AZP itself verifies it.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -133,7 +133,7 @@ stages:
     pool: sonictest
     displayName: "kvmtest-t0-part1"
     timeoutInMinutes: 360
-
+    continueOnError: true
```
Comment (SuvarnaMeenakshi) [inline azure-pipelines.yml:136]: is t0 not mandatory?

**Comment by SuvarnaMeenakshi** [comment]
https://github.com/sonic-net/sonic-buildimage/blob/master/.azure-pipelines/run-test-template.yml#L55 - can this be removed/

File: azure-pipelines.yml (hunk 1 above)
Comment (yejianquan) [reply azure-pipelines.yml:136]: t0-part 1 and t0-part2 are set as not mandatory,
kvmtest-t0 will check the results of part1 and part2, and it's mandatory

**Comment by yejianquan** [comment]
> https://github.com/sonic-net/sonic-buildimage/blob/master/.azure-pipelines/run-test-template.yml#L55 - can this be removed/

@SuvarnaMeenakshi  Yes, removed

**Review by SuvarnaMeenakshi — APPROVED** [review]

**Pull request merged**